src/MySqlConnector/Core/ResultSet.cs

```diff
@@ -55,7 +52,7 @@ public async Task ReadResultSetHeaderAsync(IOBehavior ioBehavior)
 						RecordsAffected = (RecordsAffected ?? 0) + ok.AffectedRowCount;
 						LastInsertId = unchecked((long) ok.LastInsertId);
 						WarningCount = ok.WarningCount;
-						if (ok.NewSchema is object)
+						if (ok.NewSchema != null)
```

Comment:
`is object` is preferred as a non-nullability test.

https://twitter.com/jaredpar/status/1115019017297596416